README.md

```diff
@@ -4,26 +4,40 @@
 
 ## Overview
 
-The `renv` package is a new effort to bring private project-local R libraries
-to your projects. The goal is for `renv` to be a robust, stable replacement for
-the [Packrat](https://rstudio.github.io/packrat/) package, with fewer surprises
-and better default behaviors.
+The renv package creates **r**eproducible **env**ironments that increase the
+reproducibility of your code, so it return the same results wherever and 
+whenever it is run. renv uses two tools to make this possible:
+
+* A __lockfile__, `renv.lock`, records the precise details of installed packages
+  so that anyone can reinstall exactly the same versions as you're using.
```

Comment:
Wouldn't it be better to advertise the fact that `renv::restore()` will know how to restore these packages?

Reply:
My goal here was to establish the terminology to use when explaining the tools